Dockerfile

```diff
@@ -24,7 +28,7 @@ RUN arch="$(uname -m)" && if [ "${arch}" = "aarch64" ]; then \
     elif [ "${arch}" = "x86_64" ]; then \
       yq_arch=amd64; \ 
     fi && \ 
-    curl -L "https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_${yq_arch}" -o yq && \
+    curl -L "https://github.com/mikefarah/yq/releases/download/v4.44.3/yq_linux_${yq_arch}" -o yq && \
```

Comment:
added latest version of yq due to vulnerabilities. we need to use correct yq command format to update yaml files.